src/net_processing.cpp

```diff
@@ -3831,8 +3831,13 @@ bool PeerLogicValidation::SendMessages(CNode* pto)
 
                         // add rebroadcast txns
                         pto->m_tx_relay->setInventoryTxToSend.insert(setRebroadcastTxs.begin(), setRebroadcastTxs.end());
+
+                        // also include wallet txns that haven't been successfully broadcast yet
+                        LogPrint(BCLog::NET, "Force initial broadcast of %lu transactions \n", mempool.m_unbroadcast_txids.size());
```

Comment:
this commit `38599640a1afae2bcf2aa46b0009bc0b94a46434` introduces `m_unbroadcast_txids` before it's defined anywhere.

I'd also suggest a rename if it's wallet-related only. `m_unbroadcast_wallet_txids`

Reply:
cleaned up my commits, they should be good now.

as you've mentioned, its wallet & rpc-submitted txns, so I could rename to `m_unbroadcast_local_txids`, but I don't think that helps clarify so I left as is. lmk if you disagree.